### PR TITLE
Update ports and routes to be consistent across project

### PR DESCRIPTION
**Pull request by presw**
Built on top of https://github.com/closelyy/Photos-Proxy/pull/2

https://trello.com/c/EddGNhz5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,9 +8,11 @@
     "node": ">=6.13.0"
   },
   "scripts": {
-    "start": "nodemon ./server/app.js localhost:1984"
+    "start": "node ./server/app.js",
```
Comment (DanJFox86) [inline package.json:11]: Reminds me that I need to do this.

File: server/app.js [hunk 2]
```diff
@@ -1,8 +1,12 @@
 const express = require('express');
 const path = require('path');
+const axios = require('axios');
+
+const reviewsPort = 'http://localhost:3001';
```
Comment (DanJFox86) [inline server/app.js]: This is more of a "url" than a "Port"

**Review by DanJFox86 — APPROVED** [review]
Looks like a successful conversion to use the port numbers we talked about and an addition of smart proxying routes.

File: server/app.js (hunk 2 above)
Comment (presw) [reply server/app.js]: Updated.

**Pull request merged**